sisl/atom.py

```diff
@@ -1106,6 +1106,9 @@ def __getitem__(self, key):
             return [self.orbital[o] for o in range(ol[0], ol[1], ol[2])]
         elif isinstance(key, Integral):
             return self.orbital[key]
+        elif isinstance(key, str):
+            orbs = [orb for orb in self.orbital if key.lower() in orb.name().lower()]
```

Comment:
I will change this to `key in orb.name()` since if one queries `p` orbitals one will also find polarization orbitals.
So we have to be explicit.

Also, in case none match, `None` will be returned.

Reply:
I didn't notice that, great!